csharp/ql/src/semmle/code/csharp/Assignable.qll

```diff
@@ -81,7 +81,7 @@ class AssignableRead extends AssignableAccess {
 
   pragma[noinline]
   private ControlFlow::Node getAnAdjacentReadSameVar() {
-    Ssa::Internal::adjacentReadPairSameVar(this.getAControlFlowNode(), result)
+    Ssa::Internal::adjacentReadPairSameVar(_, this.getAControlFlowNode(), result)
```

Comment:
I don't know how performance-sensitive this code is, but I'd usually add an extra parameter on the left instead of on the right to avoid a `SCAN` in callers that pass `_`.

The opposite might be optimal for the `boolean simple`, parameter: if it's on the left, then it should be quick in the caller that passes `true`, and it might (I haven't tested) be fast for other callers too if they pass `any(boolean b | b = false or b = true)` instead of `_`.

Reply:
I will run a dist-compare to check performance.